libs/api/domains/auth/src/lib/resolvers/delegation.resolver.ts

```diff
@@ -22,12 +22,12 @@ import {
   PatchDelegationInput,
   UpdateDelegationInput,
 } from '../dto'
-import { Delegation } from '../models'
+import { Delegation, MergedDelegation } from '../models'
```

Comment:
We should fixed these import in the otherway, meaning we discourage the use of `index.ts` when importing inside a app/library and should import directly from the file. Importing from barrel files can cause weird hard to debug bugs.